README.md

```diff
@@ -1,26 +1,28 @@
 rsi-schedule
 ============
 
-Update (2018)
--------------
-
-RSI removed their public schedule and replaced it with their [public roadmap](https://robertsspaceindustries.com/roadmap). It did not make sense for me to keep updating this project after that as there are no precise dates to keep track of anymore. Thank you to everyone who made use of this, and I'll see you in the 'Verse!
-
 Introduction
 ------------
 
-Cloud Imperium Games released their weekly-updated internal development schedule for Star Citizen. I thought it would be useful to access it from calendars such as Google Calendar, Apple Calendar, Yahoo! Calendar, Microsoft Outlook, and many others.
+Cloud Imperium Games released their [roadmap](hhttps://robertsspaceindustries.com/roadmap/board/1-Star-Citizen) for Star
+Citizen. I thought it would be useful to access it from calendars such as Google Calendar, Apple Calendar, Yahoo! 
```

Comment:
Since I'm not alone behind this anymore, you can replace this sentence with:

> You can now access it from calendars such as Google Calendar, Apple Calendar, Yahoo! Calendar, Microsoft Outlook, and many others.